kategory-core/src/main/kotlin/kategory/data/Option.kt

```diff
@@ -1,5 +1,8 @@
 package kategory
 
+typealias Some<A> = Option.Some<A>
```

Comment:
Why don't you just declare `Some` and `None` as top level?

Same is applicable to `Left`, `Right` and so on

Reply:
Well I think we need a consistent API with all types inside their container. For example, `Ior` and `Either` have `Left` and `Right` types, and you can call them as `Either.Right` and `Ior.Right` without conflicts, also you should be able to call `Some` as `Option.Some` or just `Some`.

Reply:
What's the point of typealiases then? Name clashing can be avoided by fully qualified imports. Long imports can be avoided by short package names :)

Reply:
@artem-zinnatullin We want commonly used types available at the top level `kategory.*` package since many users will just `import kategory.*` to obtain syntax and type class instances. Users wanting just a subset of Kategory can still import a la carte types already and that will not change.